website/docs/enterprise/getting-started/install-enterprise-single.mdx

```diff
@@ -0,0 +1,1165 @@
+---
+title: Install Weave GitOps Enterprise
+hide_title: true
+pagination_next: enterprise/getting-started/releases-enterprise
+toc_max_heading_level: 4
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import TierLabel from "@site/docs/_components/TierLabel";
+import AlphaWarning from "../../_components/_alpha_warning.mdx";
+import CurlCodeBlock from "../../_components/CurlCodeBlock";
+import oauthBitbucket from '/img/oauth-bitbucket.png';
+import oauthAzureDevOps from '/img/oauth-azure-devops.png';
+import oauthAzureDevOpsSuccess from '/img/oauth-azure-devops-success.png';
+
+# Install Weave GitOps Enterprise<TierLabel tiers="Enterprise" />
+
+:::tip
+There is no need to install the open source version of Weave GitOps before installing Weave GitOps Enterprise.
+:::
+
+You are a Platform Engineer who is in charge of installing Weave GitOps Enterprise in your organisation. We propose you
+to follow this itinerary to complete a successful installation:
+
+1. **Getting Started**: it's your day-0, and you want to get started quickly with limited knowledge for having a basic understanding on Weave GitOps Enterprise.
```

Comment:
1. Getting for day 0 with basic experinces as we designed in the cli initative ... for example, anything oidc goes to day 1